start.sh

```diff
@@ -18,15 +18,101 @@ usage() {
     echo 'that command, for testing purposes.'
 }
 
+tritonConfigured() {
+
+    # is node-triton installed?
+    local triton_installed=0
+    command -v triton >/dev/null 2>&1 && triton_installed=1
+    if [ "0" = "$triton_installed" ]; then
+        echo
+        echo "Error!"
+        echo "The Triton CLI tool does not appear to be installed!"
+        echo
+        echo "Please visit:"
+        echo "https://www.joyent.com/blog/introducing-the-triton-command-line-tool"
+        echo "for installation instructions."
+        echo
+
+        sleep 3
+        command -v open >/dev/null 2>&1 && `open https://www.joyent.com/blog/introducing-the-triton-command-line-tool` || true
```

Comment:
Is the way we're showing this error a little too over-the-top? We've got a very large error message *and* we're opening the user's browser. I get that sometimes we need to drill it in a bit, but I feel like that's going to be annoying to people.